ddp-workspace/projects/ddp-osteo/src/styles/family-history.scss

```diff
@@ -39,21 +39,39 @@
     .PageContent-text + details {
         font-weight: 600;
         margin-top: 42px;
-        padding: 18px 0 18px 10px;
+        padding: 4px 0 18px 30px;
         border-width: 1px 0;
         border-style: solid;
         border-color: #D2D2D2;
 
-        summary {
+        > summary {
+            list-style-type: none;
             outline: none;
             cursor: pointer;
+            margin-left: -16px;
+        }
+
+        > summary::-webkit-details-marker {
+            display: none;
+        }
+
+        > summary::before {
+            content: '+';
+            font-size: 30px;
+            position: relative;
+            top: 4px;
+            left: -10px;
+        }
```

Comment:
I really need to brush up on my CSS!

Reply:
Added a bit of animation there at f341478afe184ea858b17eb10be0a2e893e91d55